### PR TITLE
Refactor cosine lr scheduler

### PR DESCRIPTION
**Pull request by wyg1997**
- rename `CosineAnnealingLR` -> `CosinDecayLR`，对齐 lazy 的 CosineDecay 接口
- 实现对齐 torch 的 CosineAnnealingLR 接口

image
image

### CODE REVIEW

File: python/oneflow/compatible/single_client/nn/optimizer/cosine_decay_lr.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 from .lr_scheduler import LrScheduler
 
 
-class CosineAnnealingLR(LrScheduler):
+class CosineDecayLR(LrScheduler):
```
Comment (strint) [inline python/oneflow/compatible/single_client/nn/optimizer/cosine_decay_lr.py:21]: 看了下，这个版本对齐的是tf的，命名也没问题，

https://github.com/keras-team/keras/blob/master/keras/optimizer_v2/learning_rate_schedule.py#L549-L638

https://github.com/tensorflow/tensorflow/blob/v2.6.0/tensorflow/python/keras/optimizer_v2/legacy_learning_rate_decay.py#L454-L514

Comment (strint) [reply python/oneflow/compatible/single_client/nn/optimizer/cosine_decay_lr.py:21]: 初始化的参数接口是不是也改成torch风格的？
```
 def __init__(
        self, optimizer, steps: int, alpha: float = 0.0, last_step=-1, verbose=False
    ):
```
比如这个的steps，应该是T_max？

**Review by strint — APPROVED** [review]

File: python/oneflow/compatible/single_client/nn/optimizer/cosine_decay_lr.py (hunk 1 above)
Comment (wyg1997) [reply python/oneflow/compatible/single_client/nn/optimizer/cosine_decay_lr.py:21]: 这个原来是对齐我们 lazy 中的命名的，看了下 tf 的文档，应该叫 decay_steps 更合适一点？
